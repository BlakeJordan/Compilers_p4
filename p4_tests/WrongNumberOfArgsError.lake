int f(int a) {
    write "hehehe";
}

int g() {
    f(12, 4);
}