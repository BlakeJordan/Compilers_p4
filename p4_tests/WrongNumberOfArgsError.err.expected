6,5: Function call with wrong number of args
Type checking failed
